tests/test_trial.py

```diff
@@ -133,6 +133,32 @@ def test_suggest_uniform(storage_init_func):
         assert mock_object.call_count == 3
 
 
+@parametrize_storage
+def test_suggest_loguniform(storage_init_func):
+    # type: (typing.Callable[[], storages.BaseStorage]) -> None
+
+    with pytest.raises(ValueError):
+        distributions.LogUniformDistribution(low=1.0, high=0.9)
+
+    with pytest.raises(ValueError):
+        distributions.LogUniformDistribution(low=0.0, high=0.9)
+
+    mock = Mock()
+    mock.side_effect = [1.0, 2.0, 3.0]
+    sampler = samplers.RandomSampler()
+
+    with patch.object(sampler, "sample_independent", mock) as mock_object:
+        study = create_study(storage_init_func(), sampler=sampler)
+        trial = Trial(study, study._storage.create_new_trial(study._study_id))
+        distribution = distributions.LogUniformDistribution(low=0.1, high=3.0)
```

Comment:
It may be a nit-picky comment, but I don't think `LogUniformDistribution` includes the `high` value.
https://optuna.readthedocs.io/en/stable/reference/trial.html#optuna.trial.Trial.suggest_loguniform
```suggestion
        distribution = distributions.LogUniformDistribution(low=0.1, high=4.0)
```

Reply:
Thanks for the catch. I applied your suggestion.